lms/templates/course.html

```diff
@@ -6,34 +6,17 @@
 %>
 <%page args="course" />
 <article id="${course.id | h}" class="course">
-  %if course.is_newish:
-    <span class="status">${_("New")}</span>
-  %endif
   <a href="${reverse('about_course', args=[course.id.to_deprecated_string()])}">
```

Comment:
I have an issue with wrapping this anchor element around all of the text associated with the course.  The accessible label for this link will be title of the course, school, number, title, startdate, all read out as one big long string with no breaks or pauses.  In the past, we have included just the course title as the child of the anchor element and expanded the bounds of this element to cover the entire course card.  Alternatively, you can add an aria-hidden=true to the container(s) that wrap the school, number, title and startdate  and then create a separate list of these items that is displayed offscreen using the ```sr``` class.